plasma_framework/contracts/src/exits/payment/controllers/PaymentProcessInFlightExit.sol

```diff
@@ -123,6 +127,8 @@ library PaymentProcessInFlightExit {
             if (!success) {
                 emit InFlightBondReturnFailed(exit.bondOwner, exit.bondSize);
             }
+
+            emit InFlightExitFinalized(exitId);
```

Comment:
whoa! how was this one missed. cool we finally cought it

Reply:
Can we swap L141 and L142 🙈 if we add another commit